src/lib/util.js

```diff
@@ -2,6 +2,32 @@ import config from 'config';
 import crypto from 'crypto';
 const algorithm = 'aes-256-ctr';
 
+/**
+ * Get current store code from parameter passed from the vue storefront frotnend app
+ * @param {Express.Request} req 
+ */
+export function getCurrentStoreCode (req) {
+  if (req.headers['x-vs-store-code']) {
+    return req.headers['x-vs-store']
```

Comment:
We check for the `x-vs-store-code` and get `x-vs-store` for the headers. Is this wanted?

Reply:
Thanks this is an issue!